app/routes/admin/asset.py

```diff
@@ -23,7 +23,7 @@ async def post(self):
             # validate body
             validate(data, schema=admin_asset_handler_body_schema)
 
-            inserted_asset = await post_asset_to_postgres(data)
+            inserted_asset = await post_asset_to_postgres(data, self.session)
```

Comment:
Can we name this something like `self.db_session` instead? Because we will also have user auth sessions in the future